base/cvd/cuttlefish/host/commands/cvd/server_command/display.cpp

```diff
@@ -99,14 +98,13 @@ class CvdDisplayCommandHandler : public CvdServerHandler {
   Result<Command> HelpCommand(const RequestWithStdio& request,
                               const cvd_common::Args& subcmd_args,
                               cvd_common::Envs envs) {
-    CF_EXPECT(Contains(envs, kAndroidHostOut));
+    auto android_host_out = CF_EXPECT(AndroidHostPath(envs));
     auto cvd_display_bin_path =
-        ConcatToString(envs.at(kAndroidHostOut), "/bin/", kDisplayBin);
-    std::string home = Contains(envs, "HOME")
-                           ? envs.at("HOME")
-                           : CF_EXPECT(SystemWideUserHome());
+        ConcatToString(android_host_out, "/bin/", kDisplayBin);
+    std::string home = Contains(envs, "HOME") ? envs.at("HOME")
+                                              : CF_EXPECT(SystemWideUserHome());
     envs["HOME"] = home;
-    envs[kAndroidSoongHostOut] = envs.at(kAndroidHostOut);
+    envs[kAndroidSoongHostOut] = android_host_out;
```

Comment:
Here and in other commands:

Should this also set `envs["ANDROID_HOST_OUT"]`? Since `AndroidHostPath` can sometimes return a value when `$ANDROID_HOST_OUT` is not set, this part could work only for something later to fail.

Reply:
Done